src/test/run-make/sepcomp-inlining/Makefile

```diff
@@ -10,6 +10,6 @@ all:
 	$(RUSTC) foo.rs --emit=llvm-ir -C codegen-units=3 \
```

Comment:
I think this might break the point of the test.

Reply:
I compared the output of the test between my version and a version compiled with rust nightly. I think the unwinding stuff by the new `lang_start` moves a lot of stuff in the codegen untis around. But if you have a better idea, I'm open :)